demos/EasyChair/ProcessReviews.cpp

```diff
@@ -0,0 +1,324 @@
+#include <iostream>
```

Comment:
A README would be good; I should probably also pull this into its own repository, since I imagine it will be used by others.  EasyChair doesn't give a nice way to collate the information about papers to make final decisions, but it's not actually too hard to scrape.